code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -442,7 +442,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	descname = "Turret"
 	name = "Ocular Warden"
 	desc = "Forms an automatic short-range turret that deals low sustained damage to the unenlightened in its range."
-	invocations = list("Thne�qvnaf...", "...bs gur Ratvar...", "...qrs�raq!")
```

Comment:
You might ask 'why is that last one changed?' it had a special character in it,  (which is like a reverse ` if you can't see it) and I removed it.